public/js/practise.js

```diff
@@ -77,7 +79,7 @@ function countVowel(str) {
         }
```

Comment:
The start is correct, because you started to use **for()**, I will give you a small hint.
Replace everything from your function starting from line **74** to **79** with:

`for (let i = 0; i < str.length; ++i) {`
         `console.log(str[i]);`
`}`

-Try to see what happens in console and use this as a solution, and you can read a little about **array** in *Javascript* _link down below_
[https://www.w3schools.com/js/js_arrays.asp](url)